app/views/code.scala.html

```diff
@@ -29,9 +29,9 @@ <h1>
                 <h1>About the code</h1>
 
                 <p>Play Framework is hosted on <a href="//github.com/playframework">GitHub</a> using the Git version control system. You can get the code with the following command:</p>
-                <pre><code>git clone git://github.com/playframework/playframework.git</code></pre>
+                <pre><code>git clone git@@github.com:playframework/playframework.git</code></pre>
```

Comment:
Maybe just `https://github.com/playframework/playframework.git`? @xuwei-k WDYT?

Reply:
Makes sense to use https for new commers that don't have ssh keys set up on their machine to access GitHub. Maybe we can just add both approaches like:

```sh
$ git clone https://github.com/playframework/playframework.git

# If you have SSH keys set up associated with your GitHub account you can also use:
$ git clone git@github.com:playframework/playframework.git
```
but I am also ok to just use https since no one will push the repo directly...

Anyway, if we change it, needs to be changed here too:
https://www.playframework.com/documentation/3.0.x/BuildingFromSource#Grab-the-source

Reply:
Let's get this merged how it is. (Let's assume people know how to clone a GitHub repo and are not total noobs)

Reply:
Also better to merge now because currently it's broken...